buildSrc/src/main/java/org/candlepin/gradle/SpecVersion.java

```diff
@@ -28,9 +28,9 @@ public void apply(Project project) throws GradleScriptException {
         String versionFromSpec = null;
         String releaseFromSpec = null;
 
-        // Read the spec file from (projectRoot)/server/candlepin.spec.tmpl
```

Comment:
One more question I just realized: The buildSrc/build.gradle file was left untouched. Is there no way to merge that functionality into the main build.gradle file? (it's not a big deal if we do leave it there, but I am just curious if that was accidentally missed, or something is blocking us from removing it)

Reply:
Actually ```buildSrc/build.gradle``` is for separate build whose purpose is to build any tasks, plugins, or other classes which are to be used in main build. We use that for extending openapi, gettext, msgfmt plugin etc, that need to be compile before hand.

Reply:
ack